spec/spec_helper.rb

```diff
@@ -93,4 +93,5 @@
   # particularly slow.
   config.profile_examples = 10
 =end
+  require 'webmock/rspec'
```

Comment:
So far we've followed the pattern of putting library-specific config into individual files under `support/`, eg `support/capybara.rb` — could webmock go the same way?